docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: ~/layouts/Main.astro
+title: Comparing Astro
+---
+
+We often get asked the question, "How does Astro compare to my favorite site builder, ________?" This guide was written to help answer that question for several popular Astro alternatives.
+
+If you don't see your favorite site builder listed here, [ask us in Discord.](https://astro.build/chat)
+
+## Docusaurus vs. Astro
+
+**[Docusaurus](https://docusaurus.io/)** is a popular open source framework for building documentation websites. Docusaurus uses React for HTML generation while Astro supports HTML, Astro components and most popular web frameworks including React. 
+
+Docusaurus was designed specifically for documentation websites, and may have features for documentation websites that Astro does not. Astro was designed for any static website, which includes most documentation websites. Astro has all that you need to build fast documentation websites, including Markdown support and file-based routing.
+
+To build a documentation website with Astro, check out our official [docs](https://github.com/snowpackjs/astro/tree/main/examples/docs) starter template. This website was built using this same template!
+
+#### Comparing Docusaurus vs. Astro Performance
+
+In almost all cases, a website built with Astro will load significantly faster than a website built with Docusaurus. This is because Astro uses Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. Docusaurus websites don't support this, and instead load your entire JavaScript application in the browser. The result is a slower page load and worse Lighthouse scores than a similar website built with Astro. 
+
+Case Study: [docusaurus.io/docs](https://docusaurus.io/docs) (built with Docusaurus) loads **238kb** of JavaScript on first load, and scores a Lighthouse performance score of [26 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocusaurus.io%2Fdocs). This website, (built with Astro) loads only **9.3kb** of JavaScript (96% less) and scores a Lighthouse performance score of [94 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocs.astro.build%2Fgetting-started). 
```

Comment:
"This Website" also makes me feel like a link is missing, at first I thought that the documentation from docusaurus was recreated with Astro for comparison. Not sure what would be the best for clearing this, maybe just a link to the starting page of the docs?

Reply:
```suggestion
Case Study: [docusaurus.io/docs](https://docusaurus.io/docs) (built with Docusaurus) loads **238kb** of JavaScript on first load, and scores a Lighthouse performance score of [26 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocusaurus.io%2Fdocs). [This docs website](/), (built with Astro) loads only **9.3kb** of JavaScript (96% less) and scores a Lighthouse performance score of [94 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocs.astro.build%2Fgetting-started). 
```

Maybe this?